meegkit/trca.py

```diff
@@ -149,7 +234,14 @@ def fit(self, X, y):
                     trains[class_i, fb_i] = eeg_tmp
                 # Find the spatial filter for the corresponding filtered signal
                 # and label
-                w_best = trca(eeg_tmp)
+                if self.method == 'original':
+                    w_best = trca(eeg_tmp)
+                elif self.method == 'riemann':
+                    w_best = trca_regul(eeg_tmp, self.regul)
```

Comment:
It would be great if the riemannian method was called at least once in the test. 

```python
@pytest.mark.parametrize('ensemble', [True, False])
@pytest.mark.parametrize('method', ['original', 'riemann'])
def test_trcacode(ensemble):
```

See this file:
https://github.com/nbara/python-meegkit/blob/4aa4ba49354b996be20eda41660a550d1bd31f9a/tests/test_trca.py#L45